lib/os-cluster-entrypoint.ts

```diff
@@ -217,6 +217,14 @@ export class OsClusterEntrypoint {
         mlNodeStorage = parseInt(mlSize, 10);
       }
 
+      // This setting only imposes a requirement that Imdsv2 be enabled for the single node EC2 instance or Auto
+      // Scaling Groups of the infra-stack. If disabled the EC2 instance will follow the order of precedence, as
+      // described here: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-options.html#instance-metadata-options-order-of-precedence,
```

Comment:
It seems to me that we want to somehow explicitly specify `HttpTokens: optional` in the launch config so that we don't have to fall back to the account setting